topics/admin/tutorials/connect-to-compute-cluster/slides.html

```diff
@@ -0,0 +1,138 @@
+---
+layout: tutorial_slides
+topic_name: admin
+tutorial_name: connect-to-compute-cluster
+logo: "GTN"
+---
+layout: true
+name: left-aligned
+class: left, middle
+
+---
+
+# Why cluster?
+
+Running jobs on the Galaxy server negatively impacts Galaxy UI performance
+
+Even adding one other host helps
+
+Can restart Galaxy without interrupting jobs
+
+---
+# Cluster options
+
+- Slurm
+- Condor
+- Torque
+- PBS Pro
+- LSF
+- SGE derivatives maybe?
+- Any other [DRMAA](https://www.drmaa.org/)-supported DRM
```

Comment:
`https://www.drmaa.org/ failed` got `response code 0`

Reply:
but it is working